README.md

```diff
@@ -1,300 +1,12 @@
-# Welcome to the EPIC workshop in Trieste!
+# Welcome to the EPIC workshop!
 
 This repository serves as a tutorial to show you how to prepare, run and analyse a moist bubble
 simulation using the [Elliptical Parcel-In-Cell (EPIC)](https://doi.org/10.1016/j.jcpx.2023.100136)
-method. During the course of this workshop (July 8th 2024), we provide you temporary access to the
+method. During the course of this workshop, we provide you temporary access to the
 Cirrus cluster, a Tier-2 system of the EPSRC. All information how to access Cirrus is provided below.
```

Comment:
The phrasing we ask to be used is:

"Cirrus UK National Tier-2 HPC Service at EPCC (http://www.cirrus.ac.uk) funded by the University of Edinburgh and EPSRC (EP/P020267/1)"